rllib/core/rl_module/tests/test_marl_module.py

```diff
@@ -153,6 +153,75 @@ def test_add_remove_modules(self):
             override=True,
         )
 
+    def test_save_to_from_checkpoint(self):
+        """Test saving and loading from checkpoint after adding / removing modules."""
+        env_class = make_multi_agent("CartPole-v0")
+        env = env_class({"num_agents": 2})
+        module = DiscreteBCTorchModule(
+            config=RLModuleConfig(
+                env.observation_space,
+                env.action_space,
+                model_config_dict={"fcnet_hiddens": [32]},
+            )
+        ).as_multi_agent()
+
+        module.add_module(
+            "test",
+            DiscreteBCTorchModule(
+                config=RLModuleConfig(
+                    env.observation_space,
+                    env.action_space,
+                    model_config_dict={"fcnet_hiddens": [32]},
+                )
+            ),
+        )
+        module.add_module(
+            "test2",
+            DiscreteBCTorchModule(
+                config=RLModuleConfig(
+                    env.observation_space,
+                    env.action_space,
+                    model_config_dict={"fcnet_hiddens": [128]},
+                )
+            ),
+        )
+        with tempfile.TemporaryDirectory() as tmpdir:
+            module.save_to_checkpoint(tmpdir)
+            module2 = MultiAgentRLModule.from_checkpoint(tmpdir)
+            check(module.get_state(), module2.get_state())
+            self.assertEqual(module.keys(), module2.keys())
+            self.assertEqual(module.keys(), {"test", "test2", DEFAULT_POLICY_ID})
+
+        module.remove_module("test")
+
+        # check that after removing a module, the checkpoint is correct
+        with tempfile.TemporaryDirectory() as tmpdir:
+            module.save_to_checkpoint(tmpdir)
+            module2 = MultiAgentRLModule.from_checkpoint(tmpdir)
+            check(module.get_state(), module2.get_state())
+            self.assertEqual(module.keys(), module2.keys())
+            self.assertEqual(module.keys(), {"test2", DEFAULT_POLICY_ID})
+
+        # check that after adding a new module, the checkpoint is correct
+        module.add_module(
+            "test3",
+            DiscreteBCTorchModule(
+                config=RLModuleConfig(
+                    env.observation_space,
+                    env.action_space,
+                    model_config_dict={"fcnet_hiddens": [120]},
+                )
+            ),
+        )
+        # check that after removing a module, the checkpoint is correct
```

Comment:
copy-paste error: check that after *adding a module

Reply:
:) thanks

Reply:
done.